CHANGELOG.rst

```diff
@@ -2,6 +2,10 @@
 Changelog
 =========
 
+Version 8.4
```

Comment:
```suggestion
Version 9.0
```
Let's bump the major version. This is a backwards incompatible change for those who are constructing `IQMErrorProfile` instances themselves, i.e. code written using 8.3 will fail when using this new version.